test/functional/fixtures/tty-test.c

```diff
@@ -6,6 +6,9 @@
 #include <stdlib.h>
 #include <uv.h>
 
+// -V:STRUCT_CAST:641
+#define STRUCT_CAST(Type, obj) ((Type *)(obj))
```

Comment:
How? For the compiler it expands to the same thing it was before introducing the macros plus two pairs of parenthesis. Do not think this should be a problem. Also PVS warnings are false positives and AFAIR we agreed that goal should be zero warnings on two highest certainty levels.